src/applications/appeals/995/containers/App.jsx

```diff
@@ -153,7 +154,9 @@ export const App = ({
     loggedIn,
     formId: 'sc', // becomes "scBrowserMonitoringEnabled" feature flag
     version: '1.0.0',
-    sessionReplaySampleRate: 10,
+    // record 100% of staging sessions, but only 10% of production
+    sessionReplaySampleRate:
+      environment.vspEnvironment() === 'staging' ? 100 : 10,
```

Comment:
is this saying that if in staging record 100%, if not 10%. Ope yes it is. I just needed to say that out loud.

Reply:
Yes, we want to record all staging sessions for testing; but we need to keep production at 10% (it's not currently enabled in production)